prov/psm3/configure.m4

```diff
@@ -138,6 +138,8 @@ ifelse('
 			])
 		CFLAGS=$save_CFLAGS
 
+		have_libcuda=0
+		AS_IF([test $have_cuda -eq 1 && test $cuda_dlopen -eq 0], [have_libcuda=1])
```

Comment:
psm3 required dlopen to use cuda currently.

Reply:
Ah I interpreted the check backwards thinking psm3 could only statically link. Thanks for catching. I changed the check to make sure enable_cuda_dlopen is set to 1